api_app/services/airlock.py

```diff
@@ -88,31 +90,30 @@ def validate_request_status(airlock_request: AirlockRequest):
         return
 
 
-def get_storage_account_key(storage_client: StorageManagementClient, request_account_details: RequestAccountDetails):
-    return storage_client.storage_accounts.list_keys(request_account_details.account_rg,
-                                                     request_account_details.account_name).keys[0].value
-
-
 def get_required_permission(airlock_request: AirlockRequest) -> ContainerSasPermissions:
     if airlock_request.status == AirlockRequestStatus.Draft:
         return ContainerSasPermissions(read=True, write=True, list=True, delete=True)
     else:
         return ContainerSasPermissions(read=True, list=True)
 
 
-def get_airlock_request_container_sas_token(storage_client: StorageManagementClient,
-                                            request_account_details: RequestAccountDetails,
-                                            airlock_request: AirlockRequest):
-    account_key = get_storage_account_key(storage_client, request_account_details)
-    required_permission = get_required_permission(airlock_request)
+def get_airlock_request_container_sas_token(request_account_details: RequestAccountDetails, airlock_request: AirlockRequest):
+
+    source_blob_service_client = BlobServiceClient(account_url=get_account_url(request_account_details.account_name),
```

Comment:
I think you can remove the prefix source as there's no destination here...

Reply:
removed